packages/config/config/devices/0x0063/14296_zw3011.json

```diff
@@ -0,0 +1,153 @@
+{
+	"manufacturer": "GE/Jasco",
+	"manufacturerId": "0x0063",
+	"label": "14296 / ZW3011",
+	"description": "In-Wall Smart Toggle Dimmer",
+	"devices": [
+		{
+			"productType": "0x4944",
+			"productId": "0x3132",
+			"zwaveAllianceId": 3352
```

Comment:
this product doesn't appear in zwave alliance, so I left the ID of the product I copied from, 14295

is this ok?

Reply:
No. Remove it if it's not in the zwa.

Reply:
done